.github/workflows/size-limit.yml

```diff
@@ -6,11 +6,6 @@ on:
 jobs:
   size:
     runs-on: ubuntu-latest
-
-    strategy:
-      matrix:
-        node-version: [12.x]
```

Comment:
The node version wasn't even being used in this job. So this is probably copy pasta.